media-libs/lv2/patches/lv2-1.14.0.patchset

```diff
@@ -0,0 +1,20 @@
+From 8ab291b50f725404fb391f48203a2ed34294219c Mon Sep 17 00:00:00 2001
+From: begasus <begasus@gmail.com>
+Date: Sun, 22 Apr 2018 17:39:02 +0200
+Subject: add libs to pkgconfig
+
+
+diff --git a/lv2/lv2plug.in/ns/lv2core/lv2core.pc.in b/lv2/lv2plug.in/ns/lv2core/lv2core.pc.in
+index 96e5169..5fa859d 100644
+--- a/lv2/lv2plug.in/ns/lv2core/lv2core.pc.in
++++ b/lv2/lv2plug.in/ns/lv2core/lv2core.pc.in
+@@ -6,5 +6,5 @@ includedir=@INCLUDEDIR@
+ Name: lv2core
+ Version: @LV2CORE_VERSION@
+ Description: An audio plugin interface specification.
+-Libs:
++Libs: -L${libdir}
```

Comment:
Why do we need this if there aren't any lib to link with?

FYI: The `-L` flag tells the compiler to look for libraries specified via `-l` within the folder specified. If there aren't any `-l` flags, `-L` is unused.

Reply:
I thought I'd add it as there are some libs in the subfolders of lv2 in $libDir, if not needed it can be removed ...